Mocking.Helpers/Mocking.Helpers/NSubstitute/NSubstituteArgAnyCompletion.cs

```diff
@@ -42,6 +40,7 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
                 var mockedMethodArgumentList = token.Parent as ArgumentListSyntax;
                 var mockedMethodInvocation = mockedMethodArgumentList.Ancestors()
                                                                      .OfType<InvocationExpressionSyntax>()
+                                                                     .Where(IsSubstituteForMethod)
```

Comment:
This one isn't needed, in NSubstitute there is no method to declare that you are setting up your mock
**For** isn't part of the actual expression whereas it is the case for Moq with **Setup**
```csharp
Substitute.For<Calculator>().Add(Arg.Is(0), Arg.Is(1))
calculatorMock.Setup(calc => calc.Add(0, 1))
```

The only issue I have by not properly checking it, is that I have autocomplete suggestions also for regular methods, not just Substitute mock, but it's not a big deal